aissemble-open-inference-protocol-kserve/src/aissemble_open_inference_protocol_kserve/mappers/fastapi_mapper.py

```diff
@@ -0,0 +1,70 @@
+###
+# #%L
+# aiSSEMBLE::Open Inference Protocol::KServe
+# %%
+# Copyright (C) 2024 Booz Allen Hamilton Inc.
+# %%
+# This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+# #L%
+###
+
+from kserve import InferRequest, InferResponse, InferOutput
+
+from aissemble_open_inference_protocol_shared.types.dataplane import (
+    InferenceRequest,
+    InferenceResponse,
+    RequestInput,
+    RequestOutput,
+    Datatype,
+    TensorData,
+)
+
+
+class FastAPIMapper:
+    @staticmethod
+    def infer_request_to_inference_request(request: InferRequest) -> InferenceRequest:
+        inference_input_list = []
+        inference_output_list = []
+        # TODO Parameters need fix.
+        for input in request.inputs:
+            req_input = RequestInput(
+                name=input.name,
+                shape=input.shape,
+                datatype=Datatype[input.datatype],
+                parameters=input.parameters,
+                data=TensorData(input.data),
```

Comment:
c:
```suggestion
                data=TensorData(root=input.data),
```
This should fail testing